### PR TITLE
unix/freedos: djgpp errno.h have no ENOTSUP, so define it as "next number"

### PR DESCRIPTION
**Pull request by pohmelie**
#1956 solution(?)

This compiles just fine with djgpp.
42, cause [41](http://www.delorie.com/djgpp/doc/libc/libc_293.html) is the last one.

### CODE REVIEW

File: unix/mpconfigport_freedos.h [hunk 1]
```diff
@@ -36,3 +36,6 @@
 
 // djgpp dirent struct does not have d_ino field
 #undef _DIRENT_HAVE_D_INO
+
+// djgpp errno.h have no ENOTSUP
```
Comment (pfalcon) [inline unix/mpconfigport_freedos.h]: Usual idiom to do such things is to write such defined in #ifndef ENOTSUP. (What if next version of djgpp will define it?)

Comment (pohmelie) [inline unix/mpconfigport_freedos.h]: So, I should include errno.h in mpconfigport_freedos.h?

Comment (pfalcon) [inline unix/mpconfigport_freedos.h]: Hmm, yes, I'd rather say yes, than no.

Comment (pohmelie) [inline unix/mpconfigport_freedos.h]: @pfalcon, ok, updated.

**Comment by pfalcon** [comment]
I also overlooked "next value". Not too robust. Along the lines of discussion above, I changed that to Linux value (88). Merged, thanks!

**Pull request closed without merging**